.github/workflows/snyk-scan.yml

```diff
@@ -0,0 +1,55 @@
+name: Snyk Security Vulnerability Scan
+
+on:
+  pull_request:
+    types: [opened, edited, synchronize]
+  push:
+    tags:
+      - 'v.[0-9]+.[0-100]+.[0-100]+'
+    branches:
+      - 'main'
+
+jobs:
+  snyk_scan_test:
+    if: ${{ github.event_name == 'pull_request' }}
+    strategy:
+      matrix:
+        depsfiles: [
+          requirements_docs.txt,
+          requirements_extra.txt,
+        ]
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+
+      - name: Snyk scan for Python dependencies
+        uses: snyk/actions/python-3.7@master
+        env:
+          SNYK_TOKEN: ${{ secrets.SNYK_TOKEN }}
+        with:
+          args: --command=python3 --package-manager=pip --file=${{ matrix.depsfiles }} --skip-unresolved -d --fail-on=all
+  
+  snyk_scan_monitor:
+    if: ${{ github.event_name == 'push' }}
+    strategy:
+      matrix:
+        depsfiles: [
+          requirements_docs.txt,
+          requirements_extra.txt,
```

Comment:
`requirements_tests.txt` here too?

Reply:
Same here, only production dependencies were considered. If we want to scan dev dependencies as well we can add them.

Reply:
If it doesn't take a lot of time to scan, it won't hurt to add `requirements_tests.txt`, I guess.

Reply:
sure.